### PR TITLE
Feature/add admin routing

### PR DESCRIPTION
**Pull request by osaulenkoUA**
Added routing for dashboard
Created Layout for dashboard
Implemented component add product - not completed fully, need to add API CALL when we have fixed BE and also need to change some types of fields.

### CODE REVIEW

File: src/components/common/Button/index.tsx [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
+  // eslint-disable-next-line no-unused-vars
   onClick?: (event: MouseEvent<HTMLButtonElement>) => void;
   children: ReactNode;
   color: string;
   size?: string;
 }
 
+// eslint-disable-next-line no-unused-vars
 enum Colors {
+  // eslint-disable-next-line no-unused-vars
```
Comment (Andrii-Pinchuk) [inline src/components/common/Button/index.tsx:13]: If you set eslint settings, why do you still need this line?

File: src/store/reducers/productsSlice.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  addProduct: { ...new AddProduct({} as IProducts) },
+};
+
+const slice = createSlice({
+  name: 'Products',
+  initialState,
+  reducers: {
+    // GET_PRODUCTS: (state, action: { payload: any }) => {
+    //   state.myProducts = [];
+    // },
```
Comment (Andrii-Pinchuk) [inline src/store/reducers/productsSlice.ts:67]: Don't forget to remove the commented code before the commit.

File: src/constants/information.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  { id: 2, title: 'Правила та умови', route: Route.rules },
+  { id: 3, title: 'Доставка', route: Route.delivery },
+  { id: 4, title: 'Оплата', route: Route.payment },
+];
+
+interface IMenu {
+  id: number;
+  title: string;
+  route: string;
+  // eslint-disable-next-line no-unused-vars
```
Comment (Andrii-Pinchuk) [inline src/constants/information.tsx:28]: also remove here and in another places (if you set it correctly in eslint settings file.

**Review by Andrii-Pinchuk — CHANGES_REQUESTED** [review]

File: src/components/pages/AddProduct/index.tsx [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+              dispatch(
+                SET_FIELD({
+                  ['category']: {
+                    value: val?.value ?? '',
+                    label: val?.label ?? '',
+                    id: val?.id ?? '',
+                  },
+                }),
+              );
+            }}
```
Comment (Andrii-Pinchuk) [inline src/components/pages/AddProduct/index.tsx:74]: Move it to a separate method. It applies to all similar methods in onChange in this component.

File: src/store/reducers/productsSlice.ts (hunk 2 above)
Comment (osaulenkoUA) [reply src/store/reducers/productsSlice.ts:67]: I need it in the future.

Comment (osaulenkoUA) [reply src/store/reducers/productsSlice.ts:67]: I will work later on with Get_Products. Lets leave here

File: src/components/pages/AddProduct/index.tsx (hunk 4 above)
Comment (osaulenkoUA) [reply src/components/pages/AddProduct/index.tsx:74]: done.

File: src/components/common/Button/index.tsx (hunk 1 above)
Comment (osaulenkoUA) [reply src/components/common/Button/index.tsx:13]: it is just for interface and enum
if you know how to set no-unused-vars for interface and enum, assist me

File: src/constants/information.tsx (hunk 3 above)
Comment (osaulenkoUA) [reply src/constants/information.tsx:28]: it is just for interface and enum
if you know how to set no-unused-vars for interface and enum, assist me

**Review by Andrii-Pinchuk — APPROVED** [review]

**Pull request merged**